pkgs/build-support/dotnet/build-dotnet-module/hooks/default.nix

```diff
@@ -54,11 +55,12 @@ in
     } ./dotnet-install-hook.sh) { };
 
   dotnetFixupHook = callPackage ({ }:
+    let dep = if isDotnetTool then dotnet-sdk else dotnet-runtime; in
```

Comment:
Why is this needed?

Reply:
All dotnet tools need full sdk in order to work. I guess it's logical because usually tools are used at build-time.

Reply:
Hmm, can you maybe make the flag a bit more generic? Like renaming it to 'useSdkAsRuntime', and of course documenting it properly in dotnet.section.md